includes/admin/settings/class-settings-bootstrap.php

```diff
@@ -877,7 +877,7 @@ private static function settings_appearance() {
             array(
                 'id'      => 'show-directory-button',
                 'type'    => 'checkbox',
-                'name'    => _x( 'Show the "Directory" button.', 'settings', 'business-directory-plugin' ),
+                'name'    => _x( 'Show the "Directory" and "Return to Directory" button.', 'settings', 'business-directory-plugin' ),
```

Comment:
This makes the label a bit too long. Would you mind adding this after the checkbox instead? 'desc' should help here.